src/Utopia/Messaging/Adapter.php

```diff
@@ -55,7 +55,7 @@ protected function request(
         string $method,
         string $url,
         array $headers = [],
-        mixed $body = null,
+        ?string $body = null,
     ): string {
         $headers[] = 'Content-length: '.\strlen($body);
```

Comment:
if `$body` is `null`, `strlen($body)` is going to emit a warning. We should probably add a null check or something.